### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by Ntwali-Josue**
### In this pull request, I: 

- [x] Add RSpec as a gem to my project.
- [x] Create a Request spec file for all my controllers.
- [x] Check the following for each action:
  - If response status was correct.
  - If a correct template was rendered.
  - If the response body includes correct placeholder text.

### CODE REVIEW

**Comment by oloomoses** [comment]
Hi @Ntwali-Josue, Kindly spend a few minutes adding a descriptive readme to allow me to continue with the review. Without a descriptive readme, I am not allowed to proceed with the review.

**Comment by Ntwali-Josue** [comment]
@oloomoses Thanks for reminding me to do that, I am going to work on it.

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    before(:example) { get '/users' }
+    it 'renders the correct template' do
+      expect(response).to render_template(:index)
+    end
+    it 'responds the correct status' do
+      expect(response).to have_http_status(:ok)
+    end
+    it 'includes the right placeholder text' do
+      expect(response.body).to include('This is the user.')
+    end
```
Comment (oloomoses) [inline spec/requests/users_spec.rb:14]: - [ ] Kindly check out these tests. They are giving errors when RSpec is executed. below is the screenshot of the errors displayed. 
Screenshot from 2022-02-17 22-03-31

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @Ntwali-Josue, 

**Highlights**
- [ ] Well done on the readme :heavy_check_mark:
- [ ] Great job with the PR :heavy_check_mark:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/users_spec.rb (hunk 1 above)
Comment (Ntwali-Josue) [reply spec/requests/users_spec.rb:14]: hi @oloomoses , I think you used tests that are not updated, please pull the new changes and you won't get these errors.

Comment (Ntwali-Josue) [reply spec/requests/users_spec.rb:14]: By saying tests that are not updated, you may have used the tests which were pushed before I pushed the updated readme, when I was updating the readme that is when I also updated these tests.

**Review by oloomoses — APPROVED** [review]
Hi @Ntwali-Josue, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**